save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/SpringUtils.kt

```diff
@@ -70,6 +74,15 @@ inline fun <reified T : BaseEntity, R : BaseEntityRepository<T>> R.getByIdOrNotF
     "Not found ${T::class.simpleName} by id = $id"
 }
 
+/**
+ * Returns a [Page] of entities matching the given [Specification].
+ *
+ * @param spec can be null.
+ * @param pageable must not be null.
+ * @return never null.
+ */
+fun <T : Any> JpaSpecificationExecutor<T>.kFindAll(pageable: Pageable, spec: Specification<T>?): Page<T> = findAll(spec, pageable)
```

Comment:
Is this not used anywhere?

Reply:
Nope, but I don't want to lose it. Can be useful for future.
Will try to apply for existing code

Reply:
applied in another places: now they look more kotlinness